### PR TITLE
add CLOMonitor and Artifacthub badges to README

### PR DESCRIPTION
**Pull request by xmulligan**
Signed-off-by: Bill Mulligan <bill@isovalent.com>

Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [x] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [x] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [x] Provide a title or release-note blurb suitable for the release notes.
- [x] Thanks for contributing!

Adding CLOMonitor and Artifacthub badges to the README

```release-note
Badges for CLOMonitor and Artifacthub were added to the README
```

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -324,3 +324,11 @@ and the `2-Clause BSD License <bsd-license_>`__
 .. |cii| image:: https://bestpractices.coreinfrastructure.org/projects/1269/badge
     :alt: CII Best Practices
     :target: https://bestpractices.coreinfrastructure.org/projects/1269
+    
+.. |clomonitor| image:: https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cilium/cilium/badge
+    :alt: CLOMonitor
+    :target: https://clomonitor.io/projects/cilium/cilium
+    
+.. |artifacthub| image:: https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/cilium
+    :alt: Artifacthub
```
Comment (qmonnet) [inline README.rst]:
```suggestion
    :alt: Artifact Hub
```

**Review by qmonnet — APPROVED** [review]
Looks good! One suggestion, should we “group” the different badges by type? I mean having the two new ones next to the `cii`/`go-report` (maybe moving `slack` after that)? But that's really some bikeshedding, PR looks good if you want to merge as it is.

**Comment by xmulligan** [comment]
@qmonnet let's do that in a separate PR and merge this one as is

**Comment by qmonnet** [comment]
Sounds good (although not sure that's even worth a follow-up PR). Would you mind removing the second `Co-Authored-By:` from your commit description to appease checkpatch (and myself :stuck_out_tongue_winking_eye:), please? Feel free to remove them both, even. I haven't exactly authored the commit.

**Review by qmonnet — APPROVED** [review]

**Pull request merged**